cmd/zed/zed_conf.c

```diff
@@ -31,6 +32,12 @@
 #include "zed_log.h"
 #include "zed_strings.h"
 
+#ifdef __APPLE__
+ssize_t readv(int, const struct iovec *, int);
+ssize_t writev(int, const struct iovec *, int);
+#endif
```

Comment:
Ah, lib/libspl/include/os/macos/sys/uio.h
Nope - ZOL made the decision to not use include paths for some cmd/ entries, so that file isn't used - which is a shame. But far more work/frustration to touch their build system, then to include those prototypes.